### ISSUE TITLE
Update link to 3rd party article on Flexbox Ordering Items article

### ISSUE THREAD

**Issue by headquarters** [mdn/sprints#3361]
## URL(s)

https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Ordering_Flex_Items

## Request type

- [ ] New documentation
- [x] Correction or update

## Details

The section on accessibility and ordering (https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Ordering_Flex_Items#The_order_property_and_accessibility) has a link that is no longer related to the content. It appears the old domain is now owned by a new party and there is a new domain to link to. I believe the link to "The Responsive Order Conflict for Keyboard Focus" should be updated to https://alastairc.uk/2017/06/the-responsive-order-conflict/.

**Comment by chrisdavidmills** [comment]
Ah yes, that needs correcting. I've updated the link — thanks for reporting!
